misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/DebugExecutionManager.java

```diff
@@ -72,7 +87,12 @@ public VirtualMachine attach(String hostName, String port) throws IOException, I
         }
         connectorArgs.get(CONNECTOR_ARGS_PORT).setValue(port);
         LOGGER.info(String.format("Debugger is attaching to: %s:%s", hostName, port));
+
         attachedVm = socketAttachingConnector.attach(connectorArgs);
+        this.host = !hostName.isEmpty() ? hostName : "localhost";
```

Comment:
Shall we move `localhost` to a constant?

Reply:
70982e7